ansible/requirements.yml

```diff
@@ -8,21 +8,14 @@ collections:
 roles:
   - src: geerlingguy.docker
     version: 7.5.1
-  - src: stephdewit.nvm
-    version: v3.4.5
   - src: gantsign.default-web-browser
     version: 4.0.0
-  - src: gantsign.pin-to-launcher
-    version: 4.0.0
-  - src: bartdorlandt.starship
-    version: 0.0.5
-  - src: Oefenweb.yarn
-    version: v1.0.57
+  - src: danielrolls.nix
+    version: v3.37.0
   - src: Oefenweb.ssh_keys
     version: v5.0.5
-  - src: geerlingguy.composer
-    version: 1.9.2
   - src: andrewrothstein.dive
     version: v1.0.2
-  - src: IronicBadger.bash_aliases
-    version: 1.0.3
+  # TODO: Consider replacing custom roles with Galaxy alternatives:
+  # - robertdebock.flatpak for setup_flatpak
```

Comment:
Use this role: [alvistack.flatpak](https://galaxy.ansible.com/ui/standalone/roles/alvistack/flatpak/)